# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/html/nsIForm.h
# Commit: b9e87fde31f7
# Full Hash: b9e87fde31f7ca31185081dc96fa937927f1879f
# Author: sefeng <sefeng@mozilla.com>
# Date: 2020-05-12 04:04:10
# Regressor Bug: 1557303
# File Overlap Count: 1
# Description:
#   Bug 1557303 - Implement form[method="dialog"] r=smaug
#   
#   Spec: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#submit-dialog
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73364
# ==============================================================================

diff -r e02018dfce9e -r b9e87fde31f7 dom/html/nsIForm.h
--- a/dom/html/nsIForm.h	Tue May 12 00:11:12 2020 +0000
+++ b/dom/html/nsIForm.h	Mon May 11 22:56:13 2020 +0000
@@ -13,6 +13,7 @@
 
 #define NS_FORM_METHOD_GET 0
 #define NS_FORM_METHOD_POST 1
+#define NS_FORM_METHOD_DIALOG 2
 #define NS_FORM_ENCTYPE_URLENCODED 0
 #define NS_FORM_ENCTYPE_MULTIPART 1
 #define NS_FORM_ENCTYPE_TEXTPLAIN 2